layers/meta-balena-fsl-arm/conf/machine/nitrogen8mm-dwe.conf

```diff
@@ -0,0 +1,18 @@
+#@TYPE: Machine
+#@NAME: Boundary Devices Nitrogen8MM DWE
+#@SOC: i.MX8 MQ
+#@DESCRIPTION: Machine configuration for Boundary Devices Nitrogen8MM DWE
+#@MAINTAINER: Chris Dimich <chris.dimich@boundarydevices.com>
+
+MACHINEOVERRIDES = "nitrogen8mm:${MACHINE}"
+
+include conf/machine/nitrogen8mm.conf
+
+# Kernel configuration
+KERNEL_DEVICETREE += " \
```

Comment:
Do we need the other dtbs too for this machine?

Reply:
Correct, I removed them from here and from the boot partition too.